src/main.rs

```diff
@@ -1022,6 +1076,7 @@ async fn start_ldk() {
 		}
 	});
 
+	// TODO: remove after a few months since the new `OutputSweeper` was added in LDK v0.0.123.
```

Comment:
Hmm... actually I think we'll need migration code since outputs may remain spendable even if the user shuts down their node for a year. Can you confirm @TheBlueMatt?

Reply:
Sgtm. Thinking on it, seems more aligned with our goals to prioritize readability.